Classes/BackEnd/AbstractEventMailForm.php

```diff
@@ -668,10 +668,10 @@ protected function getHooks(): array
             if (\is_array($hookClasses)) {
                 foreach ($hookClasses as $hookClass) {
                     $hookInstance = GeneralUtility::getUserObj($hookClass);
-                    if (!($hookInstance instanceof \Tx_Seminars_Interface_Hook_BackEndModule)) {
+                    if (!($hookInstance instanceof \Tx_Seminars_Interfaces_Hook_BackEndModule)) {
                         throw new \UnexpectedValueException(
                             'The class ' . \get_class($hookInstance) . ' is used for the event list view hook, ' .
-                            'but does not implement the \\Tx_Seminars_Interface_Hook_BackEndModule interface.',
+                            'but does not implement the \\Tx_Seminars_Interfaces_Hook_BackEndModule interface.',
```

Comment:
This is breaking for the hooks. We need to move this change to the 4.0 release.

Reply:
Ok, we leave it there and I will rebase it later on the new branch.